src/app/cube/new-home/splash/splash.component.html

```diff
@@ -1 +1,27 @@
-<p>splash works!</p>
+<div class="wrapper">
+    <div class="first">
+        <h1>
+            Teach Cyber Today... <br />
+            Secure Tomorrow
+        </h1>
+        <p>
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit, 
+            sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. 
+            In iaculis nunc sed augue lacus viverra. Tempor commodo ullamcorper a lacus vestibulum. 
+            Facilisis mauris sit amet massa vitae tortor condimentum lacinia quis. 
+            Maecenas pharetra convallis posuere morbi leo urna molestie at.
+        </p>
+        <button routerLink="/browse">BROWSE 880+ LEARNING OBJECTS</button>
```

Comment:
we should try to keep everything consistent
```suggestion
        <button [routerLink]="['browse']">BROWSE 880+ LEARNING OBJECTS</button>
```